javascripts/govuk/analytics/google-analytics-universal-tracker.js

```diff
@@ -33,24 +34,25 @@
 
   // https://developers.google.com/analytics/devguides/collection/analyticsjs/pages
   GoogleAnalyticsUniversalTracker.prototype.trackPageview = function(path, title, options) {
-    var options = options || {};
+    var options;
```

Comment:
The `trackEvent` method was doing the same.

Now fixed: https://github.com/alphagov/govuk_frontend_toolkit/pull/332/files#diff-8b93af583c1a7ca5e9c48a89ebce2b38L63